website/docs/installation/fonts.mdx

```diff
@@ -23,6 +23,15 @@ This command needs to be executed as admin. The fonts are installed system wide.
 oh-my-posh font install
 ```
 
+If you're using Homebrew, you can tap the `homebrew-cask-fonts` repository and install Nerd Fonts from there instead:
```

Comment:
Can we use `Tabs` here to allow the use to switch between the methods? Just like we do in the Installation and shell selection?

Reply:
Should be implemented now.